client_plugin/drivers/shared/shared_driver.go

```diff
@@ -111,7 +147,109 @@ func (d *VolumeDriver) UnmountVolume(name string) error {
 
 // Create - create a volume.
 func (d *VolumeDriver) Create(r volume.Request) volume.Response {
-	log.Errorf("VolumeDriver Create to be implemented")
+	log.Infof("Entering VolumeDriver Create")
+	var msg string
+	var hnames []string
+
+	// Ensure that the node running this command is a manager
+	info, err := d.dockerd.Info(context.Background())
+	if err != nil {
+		msg = fmt.Sprintf("Failed to get Info from docker client. Reason: %v", err)
+		log.Warningf(msg)
+		return volume.Response{Err: msg}
+	}
+	if info.Swarm.ControlAvailable == false {
+		msg = fmt.Sprintf("This node is not a swarm manager. Shared Volume creation is only possible from swarm manager nodes.")
```

Comment:
Is it possible to list, inspect, volumes from all nodes or only on the swarm manager node?  Is it possible to create a volume via the vmdk plugin and then be able to share that volume across nodes via the shared plugin? If the datastores are the same then can't the shared plugin allow using volumes created by the vmdk plugin?